content/developer/writing_widgets/widget-apis.md

```diff
@@ -287,9 +287,10 @@ It allows you to make HTTP requests to previously defined widget backend endpoin
 Redirects user to home page.
 
 
-### goToPage(pageName)
+### goToPage(pageName, context)
 
 Redirects user to page with ID (URL alias) passed in `pageName` argument.
+The optional second argument is a new application context to be set - see [getContext](#getcontext") for context explanation.
```

Comment:
I was wondering why we have `#getcontext"` (with `"` at the end), but apparently at https://docs.cloudify.co/staging/RD-587-sites-map-context-fix/developer/writing_widgets/widget-apis/#gotopage-pagename-context it's working fine :-)